rdmd.d

```diff
@@ -713,9 +725,172 @@ private string[string] getDependencies(string rootModule, string workDir,
         exit(depsExitCode);
     }
 
+    if (!hasDependencies)
+        scope(exit) std.file.remove(rootModule);
```

Comment:
The scope here is the `if` block, right? There's no point to `scope(exit)` then because it just executes immediately, no?